### PR TITLE
DOC-303: Update README with configuration properties and installation instructions

### PR DESCRIPTION
**Pull request by m-hulbert**
This PR updates the README with configuration instructions and expands on the installation instructions. 

The installation instructions will need to be updated and Confluent Cloud added as an option once the connector is available on Confluent Hub.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,89 +1,134 @@
...
+    `docker-compose up -d`
 
-```
-curl -s -u "<your Ably API key>" "https://realtime.ably.io/sse?channel=kafka-connect-ably-example&v=1.1"
-```
+    **Note**: You can view the logs using `docker-compose logs connector`
 
-Produce some messages to the `kafka-connect-ably-example` Kafka topic using the
-`kafka-console-producer` CLI tool in the Kafka container:
+3. Once the pods have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
```
Comment (lmars) [inline README.md:45]: This should start "Once the containers have started" ("pods" is a Kubernetes specific term, and there is no Kubernetes involved here)

File: README.md [hunk 2]
```diff
@@ -1,89 +1,134 @@
...
+    message 2
+    message 3
+    EOF
+    ```
 
-id: e026fVvywAz6Il@1623496744539-2
-event: message
-data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64","channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
-```
+5. In the terminal window where you subscribed to the Ably channel, you will receive the following messages:
```
Comment (lmars) [inline README.md:61]: Probably say "you will receive messages similar to the following" (since the messages they get will be different)

**Review by lmars — APPROVED** [review]
Some minor comments, but otherwise spot on 👍

**Review by marklewin — APPROVED** [review]
LGTM (after including @lmars suggestions)

File: README.md (hunk 1 above)
Comment (m-hulbert) [reply README.md]: Good point - thank you!

File: README.md (hunk 2 above)
Comment (m-hulbert) [reply README.md]: Updated this too, thanks.

**Pull request merged**